ZeeHomeScreen/Classes/Components/Banner/BannerCellViewController.swift

```diff
@@ -107,16 +111,16 @@ class BannerCellViewController : UIViewController, ComponentProtocol, ComponentD
     
     func loadBanner() {
         
-        let adPlugin = ZPAdvertisementManager.sharedInstance.getAdPlugin()
-        adPresenter = adPlugin?.createAdPresenter(adView: self, parentVC: self)
+        adPresenter = Zee5Advertisement()
```

Comment:
better to keep it as is, need to use getAdPlugin() to make it init with the plugin config from zapp

Reply:
also adPresenter should be created with createAdPresenter, you can just cast the result of createAdPresenter() into  AdViewPresenter

Reply:
then you won't need to save the adPresenter